### PR TITLE
Update rhn charsets specs entry

### PR DESCRIPTION
**Pull request by sagaraivale**

### CODE REVIEW

File: insights/specs/sos_archive.py [hunk 1]
```diff
@@ -10,6 +10,10 @@
 
 class SosSpecs(Specs):
     blkid = simple_file("sos_commands/block/blkid_-c_.dev.null")
+    candlepin_log = first_of([
```
Comment (xiangce) [inline insights/specs/sos_archive.py]: I don't think you used the master branch as the base of this PR. The patch of #1142 is also included.

File: insights/specs/default.py [hunk 2]
```diff
@@ -478,7 +478,7 @@ def package_and_java(broker):
     rc_local = simple_file("/etc/rc.d/rc.local")
     redhat_release = simple_file("/etc/redhat-release")
     resolv_conf = simple_file("/etc/resolv.conf")
-    rhn_charsets = simple_command("/usr/bin/rhn-charsets")
+    rhn_charsets = first_of([simple_command("/usr/bin/rhn-charsets"), simple_file("database-character-sets")])
```
Comment (csams) [inline insights/specs/default.py]: The simple_file should go in sos_archive.py as
`rhn_charsets = simple_file("database-character-sets")`

Comment (bfahr) [inline insights/specs/default.py]: @sagaraivale is there a path associated with the `database-character-sets` file?

**Comment by sagaraivale** [comment]
@csams  : Make changes as per suggestions. Create New PR : https://github.com/RedHatInsights/insights-core/pull/1148 Kindly review.

**Pull request closed without merging**